server/webapp/WEB-INF/rails.new/app/presenters/api_v1/config/package_representer.rb

```diff
@@ -0,0 +1,105 @@
+##########################################################################
+# Copyright 2016 ThoughtWorks, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+##########################################################################
+
+module ApiV1
+  module Config
+    class PackageRepresenter < ApiV1::BaseRepresenter
+
+      error_representer
+
+      link :self do |opts|
+        opts[:url_builder].apiv1_admin_package_url(package_id: package.id)
+      end
+
+      link :doc do |opts|
+        'https://api.go.cd/#packages'
+      end
+
+      link :find do |opts|
+        opts[:url_builder].apiv1_admin_package_url(package_id: '__package_id__').gsub(/__package_id__/, ':package_id')
+      end
+
+      property :name, exec_context: :decorator
+      property :id,
+               exec_context: :decorator
+
+      property :auto_update, exec_context: :decorator
+  
+      property :package_repo,
+               exec_context: :decorator,
+               decorator: ApiV1::Config::RepositorySummaryRepresenter,
+               class: com.thoughtworks.go.domain.packagerepository.PackageRepository
+
+      collection :configuration,
+                 exec_context: :decorator,
+                 decorator: ApiV1::Config::PluginConfigurationPropertyRepresenter,
+                 class: com.thoughtworks.go.domain.config.ConfigurationProperty
+
+      def package
+        represented[:package]
+      end
+
+      def repository
+        represented[:repository]
+      end
+
+      def errors
+        package.errors
+      end
+
+      def name
+        package.getName
+      end
+
+      def name=(value)
+        package.setName(value)
+      end
+
+      def id
+        package.getId
+      end
+
+      def id=(value)
+        package.setId(value)
+      end
```

Comment:
You should be able to replace these with `delegate :name, :name=, :id, :id=, to: :package`